data/assets/scene/digitaluniverse/exoplanets.asset

```diff
@@ -28,7 +30,20 @@ local Object = {
     },
     Opacity = 1.0,
     Texture = {
-      File = textures .. "target-blue.png",
+      File = textures .. "target-white.png",
+    },
+    Coloring = {
+      ColorMapping = {
+        File = colormaps.Sequential2.Cool,
```

Comment:
When I tried this with the colormap, I thought that it was hard to distinguish between the color for like 1 and 2  planets, their color was too similar. I was thinking, if we know that the range of values are 1-8 then could we maybe use a qualitative color map instead? Then every value would have its own color and it would be easier to distinguish between them. But this should probably be discussed with the rest of the team.